### PR TITLE
Prepare for 3.0-0 release by fixing issues 940 930 and 877

### LINKED ISSUE #877: Document cwa-file health variables stored in data_quality_report
Add vignette documentation on cwa file health variables that are now stored in the data_quality_report.

### LINKED ISSUE #930: Handling of no non-movement period in g.calibrate
In g.calibrate we normally expect to find non-movement periods in the data.

If no non-movement is found, which means that internal object `nomovement` has length zero, it would be helpful if this could be handled efficiently. Currently we see an error about objects not being of the right size, which is not helpful for the user.

Usually the absence of non-movement indicates that the accelerometer is not worn for the entire recording and/or that the recording is too short to do any meaningful analyses. 

TO DO: Ensure that autocalibration is skipped when there is no nonmovement data.

### LINKED ISSUE #937: fix poor language in cut-points vignette
"Please let us know you if know of any cut-points we missed!"
is poor language, needs to be fixed.

### LINKED ISSUE #940: Ensure that segmentDAYSPTcrit.part5 is specified in a sensible way
A recent [google group post](https://groups.google.com/g/RpackageGGIR/c/BFf7jYLZyo4/m/xqJMIUSMBQAJ) shows that some users use qwindow in part 5 without thinking about it.

Using qwindow in part 5 without modifying segmentDAYSPTcrit.part5 is meaningless and will bias the research. I think we should:

- [x] Add clear warning to qwindow documentation that if qwindow is used in part 5 user should modify value segmentDAYSPTcrit.part5.
- [x] Add error to check_params if `segmentDAYSPTcrit.part5 = c(0, 0)`. Users need to be forced to choice whether they focus on waking hours or night time otherwise the segment estimates have no value.
- [x] Improve documentation with guidance on how this should be used.
- [x] Change default to match a typical use-case, because c(0, 0) gives the impression that it is what we recommend.

### PR DESCRIPTION
**Pull request by vincentvanhees**
It would be good to submit version 3.0-0 to CRAN next week such that we have an updated CRAN version before the GGIR training.

In preparation, this PR does:
- Improves guidance and default for argument `segmentDAYSPTcrit.part5`. Fixes #940
- Ensures that absence of non movement epochs for autocalibration can be handled. Fixes #930
- Add variable documentation for data_quality_report to the vignette. Fixes #877
- Changes default nonwear_approach to "2023" as we planned to do.
- Changes version number in NEWS file to 3.0-0 #880
- Addresses minor language error in cut-points vignette. Fixes #937

Checklist before merging:

- [x] Existing tests still work (check by running the test suite, e.g. from RStudio).
- [ ] Added tests (if you added functionality) or fixed existing test (if you fixed a bug).
- [x] Updated or expanded the documentation.
- [x] Updated release notes in `inst/NEWS.Rd` with a user-readable summary. Please, include references to relevant issues or PR discussions.
- [ ] Added your name to the contributors lists in the `DESCRIPTION` and `CITATION.cff` files.

### CODE REVIEW

**Comment by vincentvanhees** [comment]
I made one extra commit this morning as I realised that some users may set it to c(0, 0.01) to maximise the number of segments ignoring the bias it will produce. Therefore, the code now requires that one value is zero and the other one is at least 0.5 with a default of 0.9. Also, I expanded the documentation to clarify this.

File: man/GGIR.Rd [hunk 1]
```diff
@@ -831,12 +831,26 @@ GGIR(mode = 1:5,
...
-        Numeric vector or length 2 (default = c(0, 0)).
+        Numeric vector or length 2 (default = c(0, 0.9)).
         Inclusion criteria for the proportion of the segment that should be 
         classified as day (awake) and spt (sleep period time) to be considered 
-        valid. Usually, one of the two numbers is 0, and the other defines the 
-        proportion of the segment that should be classified as day or spt.}
-
+        valid. One of the two numbers should be 0, and the other defines the 
+        proportion of the segment that should be classified as day or spt, respectively.
+        The default setting would focus on waking hour
```
Comment (jhmigueles) [inline man/GGIR.Rd:839]: I think the default should be the other way around to focus on waking hour, right? the first number indicates the proportion of the segment that should be classified as awake to be included, the second number indicates the proportion of the segment that should be classified as sleep to be included. 

I just checked the default settings and I only get "sleep" segments in the clean version of the daysummary_Segments file, so maybe we need to change the default or change the documentation here.

**Review by jhmigueles — COMMENTED** [review]
Thanks! I think my main comment here is about the default value and documentation for the 
`segmentDAYSPTcrit.part5` argument.

File: man/GGIR.Rd (hunk 1 above)
Comment (vincentvanhees) [reply man/GGIR.Rd:839]: Yes, you are right. I agree that focus on PA is most logical as default.

**Pull request merged**